build/tools/mingw/officialbuild.bat

```diff
@@ -0,0 +1,95 @@
+@echo off
+
+rem Check if compiler version is present
+if "%1" == "" goto :ERR_NOPARM
+rem Check if architecture is present
+if "%2" == "" goto :ERR_NOPARM
+
+rem Initialize specifics parameters related to the current build
+call parameters.bat %1 %2
+
+echo Starting official build for %COMPNAME% ( %COMPBINDIR% )
+
+if %CLEANBUILD%==1 (
+    echo - Cleanup
+    for %%d in ( lib\%COMPNAME%_dll build\msw\%COMPNAME%_mswuddll build\msw\%COMPNAME%_mswudll utils\wxrc\%COMPNAME%_mswudll ) do (
+        if exist %WXDIR%\%%d (
+            echo    * removing %%d
+            rmdir /s /q %WXDIR%\%%d
+        )
+    )
+)
+
+rem Common part of the build command line
+set MAINCMDLINE=SHARED=1 MONOLITHIC=0 USE_WEBVIEW=1 CXXFLAGS="-fno-keep-inline-dllexport" OFFICIAL_BUILD=1 COMPILER_VERSION=%COMPVERS%
```

Comment:
```suggestion
set MAINCMDLINE=SHARED=1 CXXFLAGS="-fno-keep-inline-dllexport" OFFICIAL_BUILD=1 COMPILER_VERSION=%COMPVERS%
```

Reply:
I agree that we shouldn't use the default options, so applying this one, thanks!